template/appassessment-template.yaml

```
@@ -103,6 +110,28 @@ objects:
             terminationMessagePath: /dev/termination-log
             terminationMessagePolicy: File
             imagePullPolicy: Always
+          - name: oauth-proxy 
+            args:
+              - --https-address=:8888
+              - --provider=openshift
+              - --openshift-service-account=proxy
+              - --upstream=http://localhost:8080
+              - --tls-cert=/etc/tls/private/tls.crt
+              - --tls-key=/etc/tls/private/tls.key
+              - --cookie-secret=SECRET
+              - --openshift-sar={"resource":"services","verb":"get"}
+            image: quay.io/openshift/origin-oauth-proxy:4.1
```

```suggestion
            image: quay.io/openshift/origin-oauth-proxy:4.7
```
Use a newer version.